feat(data); Lampyrid Start (#116)
endless-sky/Endless-Sky-Delta: data/human/intro missions.txt

			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`
		conversation

> 			branch lampyrid

Reviewer: At the moment, it shows the default intro conversation if you have a Lampyrid, and if not, it shows the Lampyrid's text. 
```suggestion
			branch notlampyrid
```

Author: Thanks for pointing this out. I would prefer positive labeling/conditions, so I am going to flip the sections around instead of flipping the labelling.

> 				has "flagship model: Lampyrid Mk. II"

Reviewer: ```suggestion
				not "flagship model: Lampyrid Mk. II"
```

Author: The main conversation has been flipped now, so changing the condition is no longer necessary.

			`With some trepidation, you show the shipyard comptroller your license and registration. They look it over briefly before grunting and waving for you to continue on your way. Within minutes you liftoff for a test drive, piloting it out of the shipyard and onto a spare landing pad by the spaceport. Then you get out and look the ship, thinking about how much of a wreck it was when you first found it. You are busy polishing out a few rust spots with a piece of steel wool when an old man in grease-stained coveralls walks up.`
			`	"Nothing quite like your first time at the controls," he says.`
				goto firstStarship

> 			label lampyrid

Reviewer: ```suggestion
			label notlampyrid
```

> 			`With some trepidation, you show the shipyard comptroller your license and registration. They look it over briefly before grunting and waving for you to continue on your way. Within minutes you liftoff for a test drive, piloting it out of the shipyard and onto a spare landing pad by the spaceport. Then you get out and look over the ship, thinking about how much of a wreck it was when you first found it. You are busy checking the torque on a recalictrant nut when an old man in grease-stained coveralls walks up.`

Reviewer: ```suggestion
			`With some trepidation, you show the shipyard comptroller your license and registration. They look it over briefly before grunting and waving for you to continue on your way. Within minutes you liftoff for a test drive, piloting your Lampyrid out of the shipyard and onto a spare landing pad by the spaceport. Then you get out and look over the ship, thinking about how much of a wreck it was when you first found it. You are busy checking the torque on a recalcitrant nut when an old man in grease-stained coveralls walks up.`
```

			`	"Nothing quite like your first time at the controls," he says.`
			label firstStarship
			choice
				`	"Yeah, I'm still kind of in shock over how much money I spent."`
					to display
						not "flagship model: Lampyrid Mk. II"
					goto money
				`	"It took a lot of work, but it feels awesome!"`
					to display
						has "flagship model: Lampyrid Mk. II"
					goto hard_work
				`	"Well, I don't plan to get too attached to this one. I'll be flying something much nicer before long."`
					goto bigger
				`	"Beat it, old-timer, I've got more important things to do than chat with you."`
					decline
			label hard_work

> 			`	He grins. "It takes a lot of knowledge and skill to get a ship running. Good thing you aren't depending on a loan. It's a crime how much interest the banks charge to first-timers. It's been that way ever since when I first started out."`

Reviewer: He says you're not depending on a loan, but don't you start with one?

Author: Yes, we do start with one, although about 180k of it is gobbled up in the aforementioned fees so the player only starts with 300k. But the player doesn't actually *need* it to get the ship. They're free to return a large amount of it immediately. So this isn't a zero-debt start, it's the same debt, but with the freedom to get rid of a lot of it fairly quickly if desired. Being intended as an option for veteran players, more flexibility is beneficial. (and for my purposes, makes it more useful for quickly testing a clean pilot when I don't necessarily want to test a beginner ship at the same time)

			`	"<first> <last>?" he stops for a moment. "I've heard that name before. Reminds me of someone famous, I think," says James. "Anyways, I'm looking forward to traveling with you."`
			label advice
			`	As you're helping him wheel his luggage aboard and showing him his bunk, he says, "Before we take off, you might head on over to the trading center and stock up on medical goods; we can sell those for a good profit on <planet>. Also make sure to visit the local outfitter. While they don't sell much here compared to on the big manufacturing worlds, every world with an outfitter will at least carry a map with information on the neighboring systems. Buying them can help if you're not quite sure where you're going."`
			branch registration
				has "flagship model: Lampyrid Mk. II"
			`	With that, he settles down to wait for departure.`
				accept
			label registration

> 			`	He frowns down at his commlink then smiles "I see you haven't even finished the registration process yet. You should probably pull up your pilot information and register a name for your ship. If they haven't changed the forms since I last did it, you should just need to click on the ship tab, and then click on the blank where your ship name should be. Then you will be all legal."`

Reviewer: ```suggestion
			`	He frowns down at his commlink, then smiles. "I see you haven't even finished the registration process yet. You should probably pull up your pilot information and register a name for your ship. If they haven't changed the forms since I last did it, you should just need to click on the ship tab, and then click on the blank where your ship name should be. Then you will be all legal."`
```
Of course, there could also be players who don't do this and wonder why they don't get into trouble, but as I think Delta is mostly used by more experienced players, that's unlikely to be an issue.

Author: Well, there's nothing stopping someone from making a mission that checks for ship name = "" and giving them a rough time in some Republic port for not having finalized the registration of their ship... :)

In fact, it'd be an amusing mission to add at some point.
